o2a/converter/tasks_group.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Class for Airflow's task group"""
+# noinspection PyPackageRequirements
+from typing import List, Optional
+
+from airflow.utils.trigger_rule import TriggerRule
+
+from o2a.converter.exceptions import O2AException
+from o2a.converter.task import Task
+from o2a.converter.relation import Relation
+
+
+class TaskGroup:
+    """Class for Airflow's task group"""
```

Comment:
Maybe here we could add a few words about what such a group is and why it's needed?

Reply:
Yes, I will add. I did not want to invest in documentation during the POC period.